cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -92,13 +96,21 @@
             .grid_column(4);
         });
     }
+
     &_image {
         display: block;
         width: 100%;
     }
+
     &_title a {
         .u-link__colors( @black, @pacific-80 );
     }
+
+    &_byline,
+    &_meta .tags {
```

Comment:
Probably. Do you want that here or its own PR? It'd have to move as it's directly in includes currently.